python/ray/autoscaler/autoscaler.py

```diff
@@ -759,7 +444,6 @@ def request_resources(num_cpus=None, num_gpus=None):
     r = services.create_redis_client(
         global_worker.node.redis_address,
         password=global_worker.node.redis_password)
-    assert isinstance(num_cpus, int)
```

Comment:
I assume you removed this because it can be a float? Should we still do some input validation here?

Reply:
It can be None.